src/nxt_lib.c

```diff
@@ -87,13 +87,31 @@ nxt_lib_start(const char *app, char **argv, char ***envp)
 #ifdef _SC_NPROCESSORS_ONLN
     /* Linux, FreeBSD, Solaris, MacOSX. */
     n = sysconf(_SC_NPROCESSORS_ONLN);
+#endif
+
+#if (NXT_HAVE_LINUX_SCHED_GETAFFINITY)
+    if (n > 0) {
+        int        err;
+        size_t     setsize;
+        cpu_set_t  *set;
+
+        set = CPU_ALLOC(n);
+        if (set == NULL) {
+            return NXT_ERROR;
+        }
+        setsize = CPU_ALLOC_SIZE(n);
```

Comment:
You could consider using `size` instead and adding a blank link.
Others look good.

Reply:
Sure.